TESTING_GUIDE.md

```diff
@@ -318,3 +321,32 @@ The test runner may generate warnings that can be suppressed with `warnings:`. F
   parameters:
     index: movies
 ```
+
+## Collecting Test Coverage
+
+### Coverage Summary
+
+The test tool can generate a test coverage summary using `--coverage <path>` with the number of evaluated verb + path combinations, a total number of paths and the % of paths tested. 
+
+```json
+{
+  "evaluated_paths_count": 214,
+  "paths_count": 550,
```

Comment:
Nit: "paths_count" is ambiguous. Can we do `total_paths_count` or `exposed_paths_count`?

Reply:
Possibly, but this is existing code, so let's not scope creep it.

The spec test coverage is defined as follows.

```
export interface SpecTestCoverage {
  paths_count: number
  evaluated_paths_count: number,
  evaluated_paths_pct: number
}
```

In this context the paths count represents the spec, so adding `spec_` would duplicate these semantics.